tests/test-app/tests/apollo-cursor-pagination/knex-implementation.test.js

```diff
@@ -428,37 +494,38 @@ describe('getCatsByOwner root query', () => {
       ).toEqual([cat3, cat2].map((c) => c.id).map((id) => id.toString()));
     });
 
-    it('can sort by multiple columns', async () => {
-      await catFactory.model.query().del();
-      cat1 = await catFactory.model.query().insert({
-        ...catFactory.mockFn(),
-        id: 1,
-        name: 'Keyboard Cat 2',
-        lastName: 'C',
-      });
-      cat2 = await catFactory.model.query().insert({
-        ...catFactory.mockFn(),
-        id: 2,
-        name: 'Keyboard Cat 3',
-        lastName: 'A',
-      });
-      cat3 = await catFactory.model.query().insert({
-        ...catFactory.mockFn(),
-        id: 3,
-        name: 'Keyboard Cat 2',
-        lastName: 'B',
-      });
-      const cat4 = await catFactory.model.query().insert({
-        ...catFactory.mockFn(),
-        id: 4,
-        name: 'Keyboard Cat 1',
-        lastName: 'A',
-      });
-
-      // Ordered result set by name:asc lastName:asc should be
-      // 4A 3B 1C 2A
-
-      const query = `
+    describe('multi-column sort', () => {
+      it('can sort by multiple columns', async () => {
```

Comment:
Diff is due to added `describe` block, "hide whitespace" to make this easier to review